### ISSUE TITLE
Not an issue, but a request

### ISSUE THREAD

**Issue by smarthomelawyer** [IceBotYT/peco-outage-api#2]
I stumbled upon this integration today and was wondering, can this integration be expanded to include the Pepco outage map from the Washington DC area? 

Also, any plans on adding power usage from the Pepco app?

Good work on this!

**Comment by IceBotYT** [comment]
Hi there.
This is for PECO (Philadelphia Electric Company), not Pepco.
Sorry for the misunderstanding.

**Comment by smarthomelawyer** [comment]
No, I know. But apparently Pepco and Peco are both owned by Exelon and appear to use the same API[outage map](https://www.pepco.com/Outages/CheckOutageStatus/Pages/ViewOutageMap.aspx)_

**Comment by IceBotYT** [comment]
I see.
An entire rebranding of the integration, documentation, and library will be required. I'm not quite ready for that only a few weeks after 2022.4 released.

I hope you understand.

**Comment by stboch** [comment]
List of Codes for the 6 companies Exelon Owns. I am a user in the BGE service area interested in this as well, happy to help.
PECO - 
COMED
PEP - Pepco
BGE
DPL - delmarva
ACE - atlanticcityelectric
